webapp/Connector/src/app/store/Study.js

```diff
@@ -134,6 +158,9 @@ Ext.define('Connector.app.store.Study', {
                 return study.study_name;
             });
 
+            this.assayIdentifiers.integratedAssays = this.assayIdentifiers.map(integratedAssay => integratedAssay.assay_identifier);
```

Comment:
use a new variable instead of assigning result to this.assayIdentifiers.integratedAssays

Reply:
done